.travis.yml

```diff
@@ -63,8 +43,28 @@ jobs:
     - name: "Build documentation"
       stage: pre-test
       script: ./scripts/ci/ci_docs.sh
+    - name: "Tests postgres kubernetes python 3.6 (persistent)"
+      env: BACKEND=postgres ENV=kubernetes KUBERNETES_MODE=persistent_mode
+      python: "3.6"
+      stage: test
+    - name: "Tests postgres kubernetes python 3.6 (git)"
+      env: BACKEND=postgres ENV=kubernetes KUBERNETES_MODE=git_mode
+      python: "3.6"
+      stage: test
+    - name: "Tests postgres python 3.6"
+      env: BACKEND=postgres ENV=docker
+      python: "3.6"
+      stage: test
+    - name: "Tests sqlite python 3.5"
+      env: BACKEND=sqlite ENV=docker
+      python: "3.5"
+      stage: test
+    - name: "Tests mysql python 3.7"
+      env: BACKEND=mysql ENV=docker
+      python: "3.7"
+      stage: test
 services:
   - docker
 before_install:
   - ./scripts/ci/ci_before_install.sh
-script: "./scripts/ci/ci_run_airflow_testing.sh"
+script: travis_wait 30 "./scripts/ci/ci_run_airflow_testing.sh"
```

Comment:
I'm glad you found this parameter that I told you about.  The number 30 increases the waiting time to 30 minutes compared to 20 minutes by default.

Reply:
Wait I'm confused, why is there a 30 minute wait here?

Reply:
Something goes on in the background, but I don't write anything on the screen.  In this case, Travis stops the job. This program causes a dot to be printed on the screen from time to time, which means the program has more time to execute. I don't know the details of this situation.

Reply:
That's exactly the case. Right now in kubernetes tests we have more than 20 minutes to wait for the deployment without writing anything to output. This kills Travis builds. travis_wait is there to prolong that - side effect of travis_wait however is that it does not write ANYTHING to output until it is finished (which I found actually quite interesting - I rarely look at the output while it is running now. 

This will change in the future anyway if we switch to another build system. Now that we have kind working, I will be able to switch back to trying GitLab + Kubernetes (or maybe even Github Actions) and make it more robust and print some output while it is being built (especially after Tomek's pytest changes).

Reply:
Actually in this change I am going to optimise it a bit more and will only enable travis_wait for kubernetes tests - that make much more sense so that the other tests will not be affected.

Reply:
Done.